modules/persistent-storage-csi-snapshots-create.adoc

```diff
@@ -7,10 +7,17 @@
 
 When you create a VolumeSnapshot object, {product-title} creates a volume snapshot.
 
+
 .Prerequisites
 * Logged in to a running {product-title} cluster.
 * A PVC created using a CSI driver that supports VolumeSnapshot objects.
 * A storage class to provision the storage backend.
+* No Pods are consuming the persistent volume claim (PVC) that you want to take a snapshot of.
```

Comment:
> ! consume v
Do not use when a simpler term such as “use” would be more appropriate. Use only to mean “use up.”